daemons/mrpd/msrp.c

```diff
@@ -3625,8 +3790,18 @@ int msrp_init(int msrp_enable)
 	if (NULL == MSRP_db)
 		goto abort_socket;
 
+	if( max_interesting_stream_ids < 8 ) {
+		/* Allow minimum 8 interesting talker stream ids and 8 interesting listener stream ids */
```

Comment:
I was actually referring to the comment, which suggests that there are 16 total stream_ids, 8 talker and 8 listener.